docs/cloud-edge/modules/user-agent-filter.md

```diff
@@ -0,0 +1,127 @@
+# User Agent Filter
+
+---
+
+## Overview
+
+The User Agent Filter module allows you to `allow` or `deny` traffic based on the `User-Agent` header. You can define `allow` and `deny` regular expression rules that apply to the incoming `User-Agent` header.
+
+## Quickstart
+
+### Agent CLI
+
+```bash
+ngrok http 80 --ua-filter-allow="(foo/(\d)+.(\d)+)","(bar/(\d)+.(\d)+)" --ua-filter-deny="(buz/(\d)+.(\d)+)"
+```
+
+### Agent Configuration File
+
+```yaml
+tunnels:
+ example:
+   proto: https
+   addr: 80
+   ua_filter_allow: ["(foo/(\d)+.(\d)+)", "(bar/(\d)+.(\d)+)"]
+   ua_filter_deny: ["(buzz/(\d)+.(\d)+)"]
+```
```

Comment:
The spaces in this example seem off? `example` has one space in front, while `proto` has two